js/index.js

```diff
@@ -40,3 +40,87 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+const navigationBar = document.querySelectorAll('nav a')
+navigationBar.forEach((link, i) => {link.textContent = siteContent.nav[`nav-item-${i + 1}`]});
```

Comment:
Very nice! For anything longer I would recommend 2 lines, anything more complicated replace i, but perfect for this use case.